README.rst

```diff
@@ -37,14 +40,19 @@ on some devices. Backup your data flash before flashing!
 Installation
 -------------
 
+Installing on Ubuntu:
+^^^^^^^^^^^^^^^^^^^^^^
+Download Ubuntu-Install.sh from the Installers directory. Right click > Properties > Mark Executable.
+It will automatically download and install all of the required dependencies and evic-usb.
+
 Install from source:
 ^^^^^^^^^^^^^^^^^^^^^^
 
 Using ``evic-usb`` requires ``cython-hidapi``. You can install it using  ``pip``:
 
 ::
 
-    $ pip install hidapi
+    $ pip install hidapi click setuptools pil
```

Comment:
Remove click, setuptools and pil from this. They are installed automatically using setuptools. That line is meant for the USB dependencies